src/systems/wind_effects/WindEffects.cc

```diff
@@ -482,6 +485,10 @@ void WindEffectsPrivate::SetupTransport(const std::string &_worldName)
   // Wind info service
   this->node.Advertise("/world/" + validWorldName + "/wind_info",
                        &WindEffectsPrivate::WindInfoService, this);
+
+  // Wind info topic
+  this->wind_pub = this->node.Advertise<msgs::Wind>
+  ("/world/" + validWorldName + "/wind_info");
```

Comment:
minor lint suggestion to match gazebo coding style:

```suggestion
  this->windPub = this->node.Advertise<msgs::Wind>
      ("/world/" + validWorldName + "/wind_info");
```